src/test/scala/chisel3/tests/OptionsPassingTest.scala

```diff
@@ -0,0 +1,66 @@
+package chisel3.tests
+
+import java.io.{ByteArrayOutputStream, File, PrintStream}
+
+import chisel3._
+import chisel3.tester._
+import org.scalatest._
+import treadle.{VerboseAnnotation, WriteVcdAnnotation}
+import chisel3.tester.experimental.TestOptionBuilder._
+
+class OptionsPassingTest extends FlatSpec with ChiselScalatestTester with Matchers {
+  behavior of "Testers2"
+
+  it should "write vcd output when passing in a WriteVcdAnnotation" in {
+    // TODO: this actually relies on total thread ordering
```

Comment:
Um, this seems besides the point of the test. Why are we testing shifter behavior in a test that should be about dumping VCDs? And you don't even care about the particular stimulus, you're only looking at a particular file being generated? And does this clean up the relevant file each time, or are regressions going to be flakey?

Reply:
Um, I needed something to run that would produce a VCD a more advanced test might check that the stimulus is reflected in the vcd, but I wasn't interested in that, I mostly wanted to see that the file was created. The vcd file is 749 bytes and the test will re-use this directory, so I don't think this will be a regression problem. But I will delete to avoid confusion with successive runs.